app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/Zone.java

```diff
@@ -14,18 +16,18 @@
  */
 public class Zone {
     private List<Position> positions;
+    private static final int OVERLAY_INSIDE_COLOR = Color.argb(33, 24, 149, 244);
```

Comment:
In another situation it should!
The color cannot be accessed from the resources because this class never becomes a context and thus cannot call `getResources.getColor`  at runtime. Therefore if the color is in the `res` it needs to be fetched by an activity and passed to the instance of `Zone` which complicates a lot the process and does not really make sense because the activity does not know what it fetches the color for ..